README.md

```diff
@@ -29,6 +29,8 @@ import (
 )
 
 func main() {
+	// The format for "DATABASE_URL" is here:
+	// https://godoc.org/github.com/jackc/pgconn#ParseConfig
```

Comment:
I've been searching the wiki and README looking for this documentation. Much thanks! I was a confused because the only examples I could find were not using the standard URL format even though the variable was called `DATABASE_URL`.